webapp/src/components/ProgramActivityDialog.tsx

```diff
@@ -76,9 +76,12 @@ const ProgramActivityDialog = ({
           {timeRange()}
         </DialogContentText>
 
+        <DialogContentText><b>Program:</b>{contents.theProgram}</DialogContentText>
+
         <DialogContentText id="alert-dialog-description">
-          <b>Description:</b> {contents.description}
+          <b>Description: </b> {contents.description}
```

Comment:
Did we need a space here before the closing tag? Browsers collapse multiple spaces between elements to only a single space, so I don't think this change to this line does anything, and I think it can be reverted.

And, as before, please modify this to use `<strong>` instead. I'm sorry I didn't catch that before.